src/Java.Interop/Java.Interop/JniTypeSignature.cs

```diff
@@ -43,11 +43,13 @@ public struct JniTypeSignature : IEquatable<JniTypeSignature>
 		public JniTypeSignature (string? simpleReference, int arrayRank = 0, bool keyword = false)
 		{
 			if (simpleReference != null) {
```

Comment:
Why is `simpleReference` allowed to be `null`?

Reply:
@jonathanpeppers: it's a `struct`, and thus always has a default constructor, i.e. `new JniTypeSignature()` is always "valid" (can't/won't throw), and couldn't be overridden *at the time* (that may have since changed, but `default` provides original semantics).  Given that, and that `new JniTypeSignature()` is semantically identical to `new JniTypeSignature(null)` and `new JniTypeSignature(null, 0, false)`, why should `new JniTypeSigature(null)` throw?

*Perhaps it should* throw, but at the time it didn't make sense to me.